src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java

```diff
@@ -191,7 +191,7 @@ public boolean accept(AuditEvent event) {
         if (event.getLocalizedMessage() != null) {
             // Lazy update. If the first event for the current file, update file
             // contents and tag suppressions
-            final FileContents currentContents = FileContentsHolder.getContents();
+            final FileContents currentContents = FileContentsHolder.getCurrentFileContents();
 
             if (currentContents != null) {
```

Comment:
Coverage is failing on this line, looks like NULL is never happen.
The only way it is created is at https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java#L173
looks like no way it to be NULL, if smth be wrong during c-tor of FileContents, it will be exception, so NULL is impossible.

Please review master branch, looks like we have some hack-UT on this condition. It should be removed.